specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/examples/updateAmlCompute.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2020-03-01",
+    "parameters": {
+      "location": "eastus",
+      "properties": {
+        "computeType": "AmlCompute",
+        "properties": {
+          "description": "some compute",
+          "computeType": "AmlCompute",
+          "properties": {
+            "scaleSettings": {
+              "maxNodeCount": 4,
+              "minNodeCount": 4,
+              "nodeIdleTimeBeforeScaleDown": "PT5M"
+            }
```

Comment:
This is reported as inconsistency with the model definition.
In this object, we have two `computeType`s, one of them should be removed to align with the model.
I suppose the first should be in the wrong place, what do you think?

Reply:
I didn't touched this part, I just copy 2020-03-01 folder to this new 2020-04-01-preview folder, and add my new endpoints, examples, because ARM reviewer suggest me to add to preview version.
If that example is not correct, it is same in 2020-03-01.

Reply:
This is causing Model Validation CI's failure. Please let me get someone to fix the pipeline first.

Reply:
I would insist that this has to be fixed (maybe as well as the example in 2020-03-01), would you please have a look?
The correctness of examples is important, some of our SDK is generating the test cases from these examples (for instance python SDK). Not-working examples will block them.

Reply:
You can find the log here: https://dev.azure.com/azure-sdk/public/_build/results?buildId=374273&view=logs&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&t=43324f1f-6724-5d61-785f-f0ac9fd256a1

Reply:
fixed.